README.rst

```diff
@@ -41,7 +43,10 @@ Office/Text Document
 - office document: odt, doc, docx
 - pdf document
 
-Those file formats are generated using libreoffice. The preview generation has a fixed timeout of 60 seconds.
+Those file formats are generated using libreoffice.
+The preview generation has a default timeout of 60 seconds.
+It is possible to change this timeout by setting the `LIBRE_OFFICE_PROCESS_TIMEOUT` environment variable to any number.
+Negative numbers in this variable will disable this timeout.
```

Comment:
The standard way to disable timeout is to set 0

Reply:
according to the code, it already works, so I suggest to rewrite this sentence to `Set value to 0 (or negative) to disable the timeout.`

Reply:
Done.